task/06-conditions-n-loops-tasks.js

```diff
@@ -150,7 +184,23 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+	//  throw new Error('Not implemented');
+
+	// set variables for coordinates
+	const xc = circle.center.x,
+			yc = circle.center.y,
+			radius = circle.radius,
+			xp = point.x,
+			yp = point.y;
+
+	// calculate the distance with the Pythagorean theorem
+	const distance = Math.sqrt((xp - xc)**2 + (yp - yc)**2);
+
+	// match the distance with radius
+	if (distance < radius) {
+		return true 
+	}
+	return false;
```

Comment:
we can return exactly distance < radius, which will be converted to true or false
Please, see https://javascript.info/logical-operators

Reply:
thanks for advice!